src/components/Button.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react-native/no-unused-styles */
+import { ColorTheme } from "@/theme/colors";
+import { Space } from "@/theme/layouts";
+import { Typography } from "@/theme/typography";
+import React from "react";
+import { StyleSheet, Pressable, Text, PressableProps } from "react-native";
+
+export const Button = (
+  props: PressableProps & {
+    kind: Exclude<keyof typeof ButtonThemes, "default">;
+    style?;
+    children;
+  }
+) => {
+  const { kind, children, style, ...pressableProps } = props;
+  const ButtonStyle = StyleSheet.compose(style, ButtonThemes[kind]);
+  return (
+    <Pressable {...pressableProps} style={[ButtonStyle, ButtonLayouts.default]}>
+      <Text style={[Typography.body3, ButtonStyle]}>{children}</Text>
```

Comment:
`<Button>` should be usable as-is without having to pass children through it. We don't need to premeditate tons of customization. We just need a few easy easy components to import as-is and we can add variants as we go